### PR TITLE
#60 Use OM 1.13.2 by default instead of 1.9.4

### PR DESCRIPTION
**Pull request by rosiereflo**

### CODE REVIEW

File: dynawo/3rdParty/omcUpdate_1_13_2/utils.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  addedFile=(${SRC_OPENMODELICA}${ps}OMCompiler${ps}Compiler${ps}BackEnd${ps}XMLCreateDynawo.mo)
+
+  for patchFile in "${patchFiles[@]}"; do
+    file=`echo $patchFile | sed 's|'"${DIR_RESOURCE}"'|'"${SRC_OPENMODELICA}"'|g'`
+    patchFilesInOM+=($file)
+    patchedFile=`grep "+++" $patchFile |  awk '{print $2;}'`
+    patchedFiles+=(${SRC_OPENMODELICA}${ps}${patchedFile})
+  done
+  
+}
```
[the remark this answers is not in the document]
Comment (gautierbureau) [reply dynawo/3rdParty/omcUpdate_1_13_2/utils.sh]: It says "No newline at the end of file" in the review portal

[the remark this answers is not in the document]
Comment (rosiereflo) [reply dynawo/3rdParty/omcUpdate_1_13_2/utils.sh]: Fixed

File: dynawo/3rdParty/omcUpdate_1_13_2/XMLCreateDynawo.mo [hunk 2]
```diff
@@ -0,0 +1,2236 @@
+/*
```
[the remark this answers is not in the document]
Comment (rosiereflo) [reply dynawo/3rdParty/omcUpdate_1_13_2/XMLCreateDynawo.mo]: Fixed

File: dynawo/sources/Modeler/ModelManager/DYNModelManagerOwnFunctions.cpp [hunk 3]
```diff
@@ -206,6 +206,35 @@ void array_alloc_scalar_real_array(real_array_t* dest, int n, modelica_real firs
     va_end(ap);
 }
 
+static inline void* generic_ptrget(const base_array_t *a, size_t sze, size_t i) {
+  return (reinterpret_cast<char*>(a->data)) + (i*sze);
```
[the remark this answers is not in the document]
Comment (rosiereflo) [reply dynawo/sources/Modeler/ModelManager/DYNModelManagerOwnFunctions.cpp]: I added comments to indicate that this code is directly taken from OM

File: dynawo/sources/Modeler/ModelManager/DYNModelManagerOwnFunctions.h [hunk 4]
```diff
@@ -38,6 +38,8 @@ void copy_real_array_data_mem(const real_array_t source, modelica_real *dest);
 void array_alloc_scalar_real_array(real_array_t* dest, int n, modelica_real first, ...);
 void simple_alloc_1d_base_array(base_array_t *dest, int n, void *data);
 const char** data_of_string_c89_array(const string_array_t *a);
+void* generic_array_element_addr(const base_array_t* source, size_t sze, int dim1, ...);
```
[the remark this answers is not in the document]
Comment (rosiereflo) [reply dynawo/sources/Modeler/ModelManager/DYNModelManagerOwnFunctions.h]: Fixed

**Pull request merged**